uint64 id
geometry_msgs/Point pose
int8[] descriptor
